src/tools.ts

```diff
@@ -21,8 +22,46 @@ export interface Tool<InputSchema> {
 export interface Tools {
   docs: Tool<typeof docsInputSchema>;
   source: Tool<typeof sourceInputSchema>;
+  eval: Omit<Tool<typeof projectEvalInputSchema>, 'cli'>;
 }
 
+const projectEvalDescription = `  
+        Evaluates JavaScript/TypeScript code in the context of the project.
```

Comment:
We definitely don't want all of this leading space. So let's make sure all of the tools do not have them. For example:


```suggestion
Evaluates JavaScript/TypeScript code in the context of the project.
```